src/templates/_components/fields/input.twig

```diff
@@ -0,0 +1,54 @@
+{% from '_includes/forms' import text, textField %}
+
+{% set dimensionUnit = field.dimensionUnit %}
+{% set weightUnit = field.weightUnit %}
+
+{% if dimensionUnit %}
+    <div class="flex">
+        <div class="textwrapper">
+            {{ text({
+                id: id,
+                name: name ~ '[width]',
+                value: values.width,
+                placeholder: 'Width' | t | upper[0:1],
+                title: 'Width' | t,
+                unit: dimensionUnit,
+            }) }}
+        </div>
+        <div class="label light">/</div>
+        <div class="textwrapper">
+            {{ text({
+                id: id ~ 'height',
+                name: name ~ '[height]',
+                value: values.height,
+                placeholder: 'Height' | t | upper[0:1],
+                title: 'Height' | t,
+                unit: dimensionUnit,
+            }) }}
+        </div>
+        <div class="label light">/</div>
+        <div class="textwrapper">
+            {{ text({
+                id: id ~ 'depth',
+                name: name ~ '[depth]',
+                value: values.depth,
+                placeholder: 'Depth' | t | upper[0:1],
```

Comment:
I can't find docs on `t` and those params for `upper`. Can you share those?

Reply:
<a href="https://docs.craftcms.com/v3/dev/filters.html#translate-or-t">t or translate</a> and <a href="https://docs.craftcms.com/v3/dev/filters.html#upper">upper</a>

Reply:
Oh sorry, that upper param style is the same as <a href="https://twig.symfony.com/doc/3.x/filters/slice.html">slice</a>

Reply:
Ah I was checking Twig docs for `t`, I see it's a Craft provided filter. 👍